src/Sylius/Bundle/AdminBundle/Twig/ImpersonateCustomerExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Twig;
+
+final class ImpersonateCustomerExtension extends \Twig_Extension
```

Comment:
It's not related to impersonating customer, it's rather a `ShopExtension` or something like that.